CHANGELOG.next.asciidoc

```diff
@@ -253,6 +253,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Clone value when copy fields in processors to avoid crash. {issue}19206[19206] {pull}20500[20500]
 - Fix event.type for zeek/ssl and duplicate event.category for zeek/connection {pull}20696[20696]
 - Fix long registry migration times. {pull}20717[20717] {issue}20705[20705]
+- Adding fix to checkpoint module when logs contain time field. {pull}20567[20567]
 - Fix event types and categories in auditd module to comply with ECS {pull}20652[20652]
```

Comment:
```suggestion
- Fix event types and categories in auditd module to comply with ECS {pull}20652[20652]
- Adding fix to checkpoint module when logs contain time field. {pull}20567[20567]
```

Reply:
Please add it at the end of the CHANGELOG, should've said it before 🙇

Reply:
Ok. I will try it again and hopefully that's it then for good